api/desecapi/pdns.py

```diff
@@ -108,3 +133,18 @@ def set_dyn_records(name, a, aaaa):
             _delete_or_replace_rrset(name, 'aaaa', aaaa),
         ]
     })
+
+
+def set_rrset(zone, name, type, value):
```

Comment:
Regarding `zone` and `name`: When deleting `DS` records for `foobar.dedyn.io`, those are part of the parent zone `dedyn.io`. We thus call `set_rrset("dedyn.io", "foobar.dedyn.io", "DS", "")`.

Regarding the similarity between `set_dyn_records` and `set_rrset`: I agree that it would be nice is the former used the latter, but the two resulting requests are not desirable. Extension of `set_rrset` to `set_rrsets` is indeed the solution, but I am planning to refactor this when we generalize the API to allow general request proxying to pdns, for arbitrary record management. At that point, we'll have to rethink the design of `pdns.py` anyways.